### DISCUSSION TITLE
Get 0.45.10 classic layout in new versions

### DISCUSSION THREAD

**Discussion by zDEFz** [TriliumNext/Trilium#3005] [Ideas] [answered]
I know I might use Trilium Notes 'wrong' but I'd like to see the old layout in the new version cause I liked it better. 
![image](https://user-images.githubusercontent.com/24463722/180592208-c0e8562c-8801-40ed-abaa-3e318ce52d05.png)
Are there any plans to make it adaptable so that I am no longer stuck with the application doesnt work on my iPad when newer than 04510? Also with the space. I like the compact one... Now its a bit messy.

**Comment by zadam** [comment, marked as answer]
There are no plans from my side to support the old layout. Complaints like "too much space" can be fixed with a custom theme rather easily, though.
